apps/marketplace/components/SellerSelect/SellerSelect.scss

```diff
@@ -35,6 +35,22 @@
     padding: 0.5rem 0;
   }
 
+  .sellerSelectView {
+    margin-bottom: 1.5rem;
```

Comment:
This appears to be adding space above the list of sellers.
![Screen Shot 2019-05-28 at 9 57 56 am](https://user-images.githubusercontent.com/2645932/58442814-1eb3ef00-8131-11e9-8eae-767a87a96ffb.png)

Reply:
It probably needs to be applied to another element so we put space between the selector and the list of sellers that will be invited.
![Screen Shot 2019-05-28 at 10 14 43 am](https://user-images.githubusercontent.com/2645932/58442851-6e92b600-8131-11e9-9b64-53a2e6444ca8.png)

Reply:
fixed